portal-web/test/functional/com/liferay/portalweb/macros/headless/headlessadmintaxonomy/TaxonomyCategoryAPI.macro

```diff
@@ -13,7 +13,20 @@ definition {
 			}
 		''';
 
-		var curl = JSONCurlUtil.post("${curl}", "$.id");
+		var curl = JSONCurlUtil.post("${curl}");
+
+		return "${curl}";
+	}
+
+	macro _curlTaxonomyCategory {
```

Comment:
I feel like renaming to `_curlTaxonomyCategoryByErc` would better name for this specific curl call since `/by-external-reference-code` is hardcoded. If it was more flexible like `TaxonomyVocabularyAPI._curlTaxonomyVocabulary` then current name would make sense.